src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphAuthDAO.scala

```diff
@@ -12,6 +12,15 @@ import scala.collection.JavaConversions._
 
 class GraphAuthDAO extends AuthDAO with GraphDAO {
 
+  override def createUser(rawlsUser: RawlsUser, txn: RawlsTransaction): RawlsUser = txn withGraph { db =>
+    getUserVertex(db, rawlsUser) match {
+      case Some(_) => throw new RawlsException("Cannot create user %s in database because it already exists".format(rawlsUser))
+      case None =>
+        saveObject[RawlsUser](rawlsUser, addVertex(db, VertexSchema.User), None, db)
+        rawlsUser
+    }
+  }
+
```

Comment:
Save works like other Graph saves - meaning either create or update.

Create is for the case where we know the user or group should not be present, and therefore it throws an error if it finds them.